test/functionalTest/cases/1048_subscription_cache/csub_update_with_notif.test

```diff
@@ -0,0 +1,447 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Note this .test is in the case/1048_XXXX (not in cases/1328_XXX), due to is a test to check the ""Fix: csub cache sync logic not persisting lastNotification when the field doesn't initially exist in DB".